bdb/rep.c

```diff
@@ -1724,6 +1724,8 @@ void net_newnode_rtn(netinfo_type *netinfo_ptr, char *hostname, int portnum)
 
 /* Timestamp of when our coherency lease expires on replicant */
 uint64_t coherency_timestamp = 0;
```

Comment:
The 'dump_zero_coherency_timestamp' instrumentation was put in place as an attempt to see and understand why cluster nodes would stay incoherent for semi-extended periods of time after a partition heals.  The changes (mostly bug-fixes) in this PR have reduced the mean time to coherency to about 15 seconds on my laptop computer.  We can do better.  If we choose to work more on this, I suspect election is the next hurdle.